### PR TITLE
app-misc/pdfpc: Bumpt to 4.4.0 and take over maintenance

### PR DESCRIPTION
**Pull request by holgersson32644**

### CODE REVIEW

File: app-misc/pdfpc/pdfpc-4.4.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+src_prepare() {
+	cmake_src_prepare
+
+	sed -i -e "s/valac/valac valac-0.46 valac-0.44 valac-0.40 valac-0.36/" cmake/vala/FindVala.cmake || die
+	vala_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DSYSCONFDIR="${EPREFIX}/etc"
```
Comment (juippis) [inline app-misc/pdfpc/pdfpc-4.4.0.ebuild]:
```
 * Messages for package app-misc/pdfpc-4.4.0:

 * One or more CMake variables were not used by the project:
 *   SYSCONFDIR
```

File: app-misc/pdfpc/pdfpc-4.4.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+IUSE="+gstreamer"
+
+RDEPEND="app-text/poppler:=[cairo]
+	dev-libs/glib:2
+	dev-libs/libgee:0.8
+	gnome-base/librsvg
+	gstreamer? ( media-libs/gstreamer:1.0
+		media-libs/gst-plugins-base:1.0 )
+	sys-apps/dbus
+	x11-libs/gtk+:3"
```
Comment (juippis) [inline app-misc/pdfpc/pdfpc-4.4.0.ebuild]: Unconditional deps before conditional ones please. You could multiline gstreamer to make it more readable.

File: app-misc/pdfpc/pdfpc-4.4.0.ebuild (hunk 1 above)
Comment (holgersson32644) [reply app-misc/pdfpc/pdfpc-4.4.0.ebuild]: Fixed.

File: app-misc/pdfpc/pdfpc-4.4.0.ebuild (hunk 2 above)
Comment (holgersson32644) [reply app-misc/pdfpc/pdfpc-4.4.0.ebuild]: Fixed. I also had overseen some slot deps that I added aswell.

File: app-misc/pdfpc/pdfpc-4.4.0.ebuild [hunk 3]
```diff
@@ -4,9 +4,9 @@
 EAPI=7
 
 VALA_MIN_API_VERSION="0.34"
-VALA_MAX_API_VERSION="0.42" # fix sed line if you increase this
+VALA_MAX_API_VERSION="0.46" # append versions in in sed line if increased
```
Comment (a17r) [inline app-misc/pdfpc/pdfpc-4.4.0.ebuild:7]: in in

File: app-misc/pdfpc/pdfpc-4.4.0.ebuild [hunk 4]
```diff
@@ -17,27 +17,30 @@ SLOT="0"
...
-		media-libs/gst-plugins-base:1.0 )
 	sys-apps/dbus
-	x11-libs/gtk+:3"
+	x11-libs/gtk+:3
+	gstreamer? (
+		media-libs/gstreamer:1.0=
+		media-libs/gst-plugins-base:1.0=
+	)
+"
+
```
Comment (a17r) [inline app-misc/pdfpc/pdfpc-4.4.0.ebuild:32]: that emptyline is unnecessary

File: app-misc/pdfpc/pdfpc-4.4.0.ebuild [hunk 5]
```diff
@@ -17,27 +17,30 @@ SLOT="0"
...
 DEPEND="${RDEPEND}"
 BDEPEND="$(vala_depend)"
-
```
Comment (a17r) [inline app-misc/pdfpc/pdfpc-4.4.0.ebuild:30]: but this one really should be kept.

**Pull request closed without merging**